src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colour.values().length);
+        return String.valueOf(Colour.values()[index]);
```

Comment:
use name() method to get string representation of a enum

Reply:
Corrected